diff --git a/data/me.iepure.devtoolbox.desktop.in.in b/data/me.iepure.devtoolbox.desktop.in.in
index 74fad3b..2eae022 100644
@@ -5,9 +5,10 @@
 [Desktop Entry]
 # TRANSLATORS: do not translate
 Name=@app-name@
+Comment=Development tools at your fingertips
 Exec=devtoolbox
 Icon=me.iepure.devtoolbox
 Terminal=false
 Type=Application
-Categories=GTK;
+Categories=Development;GTK;
 StartupNotify=true
diff --git a/data/me.iepure.devtoolbox.metainfo.xml.in b/data/me.iepure.devtoolbox.metainfo.xml.in
index e47e283..ddbcbf2 100644
--- a/data/me.iepure.devtoolbox.metainfo.xml.in
+++ b/data/me.iepure.devtoolbox.metainfo.xml.in
@@ -38,6 +38,7 @@ SPDX-License-Identifier: CC0-1.0
   <url type="homepage">https://github.com/aleiepure/devtoolbox</url>
   <url type="bugtracker">https://github.com/aleiepure/devtoolbox/issues</url>
   <url type="vcs-browser">https://github.com/aleiepure/devtoolbox</url>
+  <url type="translate">https://hosted.weblate.org/engage/devtoolbox/</url>
   <url type="contribute">https://github.com/aleiepure/devtoolbox/wiki/Contributing</url>
   <screenshots>
     <screenshot type="default">
@@ -53,15 +54,14 @@ SPDX-License-Identifier: CC0-1.0
       <image>https://raw.githubusercontent.com/aleiepure/devtoolbox/main/data/appstream/4.png</image>
     </screenshot>
   </screenshots>
-  <translation type="gettext">me.iepure.devtoolbox</translation>
+  <translation type="gettext">devtoolbox</translation>
   <content_rating type="oars-1.1"/>
   <recommends>
     <control>pointing</control>
     <control>keyboard</control>
   </recommends>
   <requires>
     <display_length compare="ge">700</display_length>
-    <display_width compare="ge">1050</display_width>
   </requires>
   <releases>
     <release version="1.1.1" date="2023-06-18">
--
libgit2 1.7.2

